src/lib.rs

```diff
@@ -507,41 +560,90 @@ impl IntoRawFd for CANSocket {
 
 impl Drop for CANSocket {
     fn drop(&mut self) {
-        self.close().ok(); // ignore result
+        self.close()
+            // .map_err(|e|eprintln!("CANSocket close error: {}", e.to_string()))
```

Comment:
Left over comment

Reply:
It seems to my that a lib shouldn't spout error messages on its own, but I'm not really sure if just ignoring an error, even on drop, is wise either.